### PR TITLE
ci: add labeler action

### PR DESCRIPTION
**Pull request by HarshCasper**
This PR:

- Adds PR size labeler action
- Adds service-specific & area-specific labels action for PRs

### CODE REVIEW

**Comment by HarshCasper** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by HarshCasper** [comment]
recheck

File: .github/labeler.yml [hunk 1]
```diff
@@ -0,0 +1,112 @@
+aws:apigateway:
+  - localstack/aws/api/apigateway/*.py
+  - localstack/services/apigateway/*.py
+  
+aws:lambda:
+  - localstack/aws/api/awslambda/*.py
+  - - localstack/services/awslambda/*.py
```
Comment (whummer) [inline .github/labeler.yml:7]:
```suggestion
  - localstack/services/awslambda/*.py
```

Comment (whummer) [reply .github/labeler.yml:7]: nit: I'm assuming the double dashes were not added on purpose..?

File: .github/labeler.yml [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+  
+aws:cloudformation:
+  - localstack/aws/api/cloudformation/*.py
+  - localstack/utils/cloudformation/*.py
+  - - localstack/services/cloudformation/*.py
```
Comment (whummer) [inline .github/labeler.yml:12]:
```suggestion
  - localstack/services/cloudformation/*.py
```

File: .github/labeler.yml [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+  - localstack/services/cloudwatch/*.py
+  
+config:
+  - localstack/aws/api/config
+  - localstack/services/configservice/*.py
+  
+aws:dynamodb:
+  - localstack/aws/api/dynamodb/*.py
+  - localstack/aws/api/dynamodbstreams/*.py
+  - localstack/services/apigateway/*.py
```
Comment (whummer) [inline .github/labeler.yml:26]:
```suggestion
  - localstack/aws/api/dynamodbstreams/*.py

```

File: .github/labeler.yml [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+  
+aws:stepfunctions:
+  - localstack/aws/api/stepfunctions/*.py
+  - localstack/services/stepfunctions/*.py
+  
+aws:sts:
+  - localstack/aws/api/sts/*.py
+  - localstack/services/sts/*.py
+  
+aws:kinesis:
```
Comment (whummer) [inline .github/labeler.yml:93]: nit: Could be moved further up, to preserve the alphabetical order of service names.

**Review by whummer — APPROVED** [review]
This looks great @HarshCasper  - love it how we're increasingly adding automation to the repo! 🚀  Only a few nitpick suggestions..

**Pull request merged**